egs/aspire/s5/local/multi_condition/get_reverberate_parameter_lists.py

```diff
@@ -75,17 +67,10 @@ def exists_wavfile(file_name):
         raise Exception('Unknown format of ' + file)
       impulse_noise_index.append([impulses_set, noises_list])
 
-  
-  if params.num_files_per_job is None:
-    lines_per_file = len(wav_files)
-  else:
-    lines_per_file = params.num_files_per_job
-  num_parts = int(math.ceil(len(wav_files)/ float(lines_per_file)))
-  indices_per_file = map(lambda x: xrange(lines_per_file * (x-1), lines_per_file * x), range(1, num_parts))
-  indices_per_file.append(xrange(lines_per_file * (num_parts-1), len(wav_files)))
+  lines_per_file = len(wav_files)
+  indices_per_file = map(lambda x: xrange(lines_per_file * (x-1), lines_per_file * x), range(1, 1))
+  indices_per_file.append(xrange(0, len(wav_files)))
```

Comment:
with the new changes using nested for loops with indices_per_file and indices does not make any sense. Use a single for loop with range(len(wav_files)).